projects/angular-calendar/src/modules/common/keydown.directive.ts

```diff
@@ -0,0 +1,100 @@
+import {
```

Comment:
I think this directive can just be removed in favour of `(keydown.Enter)`

Reply:
@vandents how come this is needed over just `keydown.Enter`? The custom click directive was to handle issues with hammerjs, but that's just touch events, so shouldn't impact any keyboard events

Reply:
Initially I used `keydown.Enter` in place of the `keydown directive`. However, many things aren't key-able just using `keydown.Enter`. The enter key was being overridden (I'm not sure why), preventing some links and buttons from being enter key-able. This is the only solution I found where everything is consistently enter key-able.

There was also a problem I ran into where screen readers would hijack the enter key event and try to (unsuccessfully) execute a click event. The `keydown directive` seems to fix this problem as well.

Reply:
Oh, how weird, OK lets simplify the directive then to the bare minimum we need for it to function.